### ISSUE TITLE
Could  mongodb-grafana remove the proxy server?

### ISSUE THREAD

**Issue by hmw710** [JamesOsgood/mongodb-grafana#104]
I want to let this plugin without proxy. In other words, how can the plugin connect the mongo directly?

Do you have any idea? Can you give me some tips?  Thank you very much.

**Comment by numbpun** [comment]
I don't believe you can directly connect without using proxy since there is no support from Grafana open source. I tried to connect it directly but did not find any way to present my data graphically.
